setup.py

```diff
@@ -18,6 +18,58 @@ def find_package_data(pkg, data_paths):
     return {pkg: data}
 
 
+def load_requirements(*requirements_paths):
```

Comment:
there's a shorter version of this method in other packages, we shouldn't be repeating this everywhere